source/tyr/tyr/binarisation.py

```diff
@@ -1233,6 +1236,46 @@ def gtfs2s3(self, instance_config, filename, job_id, dataset_uid):
     _inner_2s3(self, "gtfs", instance_config, filename, job_id, dataset_uid)
 
 
+@celery.task(bind=True)
+def poi2asgard(self, instance_config, filename, job_id, dataset_uid):
+    """Extract excluded zones and synchronize with"""
+    job = models.Job.query.get(job_id)
+    dataset = _retrieve_dataset_and_set_state("poi", job.id)
+    instance = job.instance
+    logger = get_instance_logger(instance, task_id=job_id)
+
+    excluded_zone_dir = "excluded_zones"
+    if os.path.isdir(excluded_zone_dir):
+        shutil.rmtree(excluded_zone_dir)
+
+    os.mkdir(excluded_zone_dir)
+    poi_to_excluded_zones(filename, excluded_zone_dir, instance.name)
+
+    try:
+        with collect_metric("poi2Asgard", job, dataset_uid):
+            asgard_bucket = current_app.config.get('MINIO_ASGARD_BUCKET_NAME', None)
+            if not asgard_bucket:
+                raise Exception("Asgard Bucket is None")
+
+            bash_command = (
+                "env REQUESTS_CA_BUNDLE=/etc/ssl/certs/ca-certificates.crt "
```

Comment:
even though it's corrected here: https://github.com/hove-io/navitia/pull/4212/files#diff-9677996dffe09856de00ff00f281b66e799a3187047cbb83bf62efe46d434186R30,  I still have to put the env var here and I have absolutely no idea why... otherwise the `aws sync` wouldn't work...